homeassistant/components/livisi/coordinator.py

```diff
@@ -34,34 +36,25 @@ def __init__(
             name="Livisi devices",
             update_interval=timedelta(seconds=DEVICE_POLLING_DELAY),
         )
-        self.config_entry: ConfigEntry = config_entry
-        self.hass: HomeAssistant = hass
-        self.aiolivisi: AioLivisi = aiolivisi
-        self.websocket: Websocket = Websocket(aiolivisi)
-        self.devices: list[dict[Any, Any]] = []
+        self.config_entry = config_entry
+        self.hass = hass
+        self.aiolivisi = aiolivisi
+        self.websocket = Websocket(aiolivisi)
         self.switch_devices: list = []
```

Comment:
Could there be other types of devices than switches? In that case I suggest we rename this attribute and store all device representations in this container.